scripts/check-lockfiles.mjs

```diff
@@ -8,10 +8,10 @@ import path from 'node:path'
 import fs from 'node:fs'
 import { fileURLToPath } from 'node:url'
 
-const LOCKFILE_VERSION_TARGET = 2
+const LOCKFILE_VERSION_TARGET = 3
```

Comment:
As we make the move to recommend Node 18 as the only supported version, we updated lockfileVersions to 3 for all packages.
Node 18 with NPM 9 uses lockfile version 3 which is backwards compatible with npm 7.

So we update the check-lockfile script to be aligned with these changes.

Reply:
Will this break users who are still using npm 7/8?

Reply:
As @vmarta suggested, package-lock files are going to be seen by monorepo developers only. They’re not published to npm. So we don’t think there would be a major problem.